### ISSUE TITLE
About custom mfrc board project.

### ISSUE THREAD

**Issue by Mali2702** [palcan/Custom-MFRC522-Board-Design#1]
Hello,
I am a student of electric and electronic engineering. And ı really liked your project. I would want to make a familiar project. Could you tell me the dimensions you used when making the PCB layout?
thank you.
Mehmet Ali Kılıc

**Comment by Mali2702** [comment]
And if you can say how you made the routing it would be perfect for me

**Comment by palcan** [comment]
Yeah sure - I've added the Gerbers to the repo. Just use any gerber viewer you want to find the dimensions you need
